src/ebonite/ext/flask/server.py

```diff
@@ -75,31 +75,32 @@ class FlaskConfig(Config):
     FlaskConfig.log_params()
 
 
+PREBUILD_PATH = current_module_path('prebuild')
+BASE_IMAGE_TEMPLATE = 'zyfraai/flask:{}'
```

Comment:
i propose clearer name smth like `ebonite-flask-server`

Reply:
I decided to make prebuilt images fully Ebonite-agnostic (name and internals) to promote their usage in other contexts. For DockerHub users they should look like alternative to FastAPI images prebuilt by FastAPI author.

Reply:
ok, but still just "flask" looks like you just installed flask over python image. at least "flask-server"? also, to promote use of it in other context, need some documentation (like that you need app.py file or something?)

Reply:
For me it's ok not to use word "server" here. Moreover, it's somewhat consistent with "uvicorn-gunicorn-fastapi". It would be even more consistent if we have "uwsgi-nginx-flask" but "uwsgi-nginx" are implementation details and can be omitted.

For me the lack of documentation actually seems like much more important problem here. You are right, `app.py` should be replaced in order to use this image. I need to take some time to prepare documentation for the repo on Dockerhub. But I don't think it should block this PR somehow. I will notify you privately when docs are ready.